src/migrations/1584604583726_blocks.ts

```diff
@@ -75,11 +75,6 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
   });
 
   pgm.createIndex('blocks', 'block_hash');
-  pgm.createIndex('blocks', 'burn_block_hash');
-  pgm.createIndex('blocks', 'burn_block_height');
-  pgm.createIndex('blocks', 'index_block_hash');
```

Comment:
Are the unused indexes being reported by an instance that watches both read and write queries? For example, `index_block_hash` is queried in a few places during ingestion and re-org handling. It's often along-side other columns like
```sql
        FROM blocks
        WHERE block_height = $1 AND index_block_hash = $2
```
So maybe the query planner just isn't using the index for this column?

Reply:
Similarly, does the tool ever suggest missing _composite_ indexes? Like in this case, it could potentially be a net gain if there was a `(block_height, index_block_hash)` index?

Reply:
> Are the unused indexes being reported by an instance that watches both read and write queries?
Yes, PgHero shows a report for primary and read-replicas

> Similarly, does the tool ever suggest missing composite indexes?
Not AFAIK... I haven't found a section that suggests new indices, but we could definitely try that index if you think so